@here/harp-mapview/lib/Utils.ts

```diff
@@ -381,34 +423,33 @@ export namespace MapViewUtils {
     }
 
     /**
-     * Sets the rotation of the camera according to yaw and pitch in degrees. The computations hinge
-     * on the current projection and `geoCenter`, because yaw and pitch are defined in tangent
-     * space. In particular, `MapView#geoCenter` needs to be set before calling `setRotation`.
+     * Computes the rotation of the camera according to yaw and pitch in degrees. The computations
+     * hinge on the current `projection` and `geoCenter`, because yaw and pitch are defined in
+     * tangent space.
      *
      * **Note:** `yaw == 0 && pitch == 0` will north up the map and you will look downwards onto the
      * map.
      *
-     * @param mapView Instance of MapView.
+     * @param projection Current projection.
+     * @param geoCenter Current geoCenter.
      * @param yawDeg Yaw in degrees, counter-clockwise (as opposed to azimuth), starting north.
      * @param pitchDeg Pitch in degrees.
      */
-    export function setRotation(mapView: MapView, yawDeg: number, pitchDeg: number) {
-        const transform = {
-            xAxis: space.x,
-            yAxis: space.y,
-            zAxis: space.z,
-            position: cache.vector3[0]
-        };
-        mapView.projection.localTangentSpace(mapView.geoCenter, transform);
+    export function getCameraRotation(
```

Comment:
Isn't it a more general `getRotation` or `getRotationAtCoordinates` now? It doesn't deal with any object like the camera anymore. Also in this mindset, the argument `geoCenter` could be renamed `coordinates`.

Reply:
Yeah true, so I will rename `geoCenter` to `target` and the method `getCameraRotationAtTarget`.